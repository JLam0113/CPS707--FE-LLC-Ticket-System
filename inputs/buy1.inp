login
admin
buy
event1
admin
1
yes
exit
