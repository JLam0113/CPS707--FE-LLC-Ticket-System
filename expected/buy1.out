Welcome, please enter a command.
Please enter your username
Privileged Login successful, now logged in as admin, please enter a command
Buy tickets selected, please enter the event title:
Please enter the number of tickets:
Please enter the seller's username:
You are buying 1 tickets for event1 from admin for $10, can you please confirm with yes/no?
Transaction confirmed, please enter a command
Session ended
